Tests/Fixtures/SecurityTestController.php

```diff
@@ -28,11 +28,9 @@ public function scopeAction(): Response
 
     public function rolesAction(): Response
     {
-        $roles = $this->get('security.token_storage')->getToken()->getRoles();
-
-        $roles = array_map(function (Role $role): string {
-            return $role->getRole();
-        }, $roles);
+        /** @var TokenStorage $tokenStorage */
+        $tokenStorage = $this->get('security.token_storage');
```

Comment:
There is no need to get the token storage in this way. Use constructor or method injection instead,

Reply:
i'm not sure how to achieve this, so feel free to give me a code snippet or a pull request into my branch.

Reply:
You can typehint `TokenStorageInterface` in the constructor or as a param for `rolesAction` method so that there is no need to call `$this->get()` here.

Reply:
ah ok cool, yeah that worked, thank you! :)

Reply:
👍